MobilityData/mobility-feed-api
# feat: setup Google Analytics tags on UI

**Summary:**

Add Google Analytics to the pageview event per environment:
- Dev/local/test/cypress: no GA tag added to the page
- QA: GA tag added reporting to qa.mobilitydatabase.org site
- PROD: GA tag added reporting to mobilitydatabase.org site

Closes #311 

**Expected behavior:** 

Google Analytics requests are visible from GA dashboard.

**Testing tips:**

GA requests are sent to the analytics dashboard
- Deploy to QA from this branch
- Open https://qa.mobilitydatabase.org
- Open developer console
- Refresh the page and verify that the request with URL _https://www.googletagmanager.com/gtag/js?id={GA_ID}_ URL is visible on the network tab
- Go to the Google Analytics dashboard and verify that requests are generating statistics

Local 
- Start the web-app locally, `yarn start:dev`
- Open developer console
- Refresh the page and verify that the request with _https://www.googletagmanager.com/gtag/js?id={GA_ID}_ is URL **not** visible on the network tab

QA GA report:
<img width="949" alt="Screenshot 2024-02-28 at 1 19 50 PM" src="https://github.com/MobilityData/mobility-feed-api/assets/1192523/37b35bca-057e-4495-8cee-2064cb087cb6">
